src/function.ts

```diff
@@ -75,6 +86,19 @@ export class Function<P, O> implements Integration {
       ResultSelector: "$.Payload",
     };
   }
+
+  public eventBus: EventBusTargetIntegration<
```

Comment:
should this be readonly?

Reply:
Does this show up as a function in the IDE or a method?

Reply:
It does show up, yeah. I'd like for it not to, however, we now use the type to constrain the `pipe` function.